css/fullcalendar.scss

```diff
@@ -156,18 +156,65 @@
 	}
 
 	// Reminder icon on events with alarms set
-	.fc-event-nc-alarms {
-		.icon-event-reminder {
```

Comment:
The notification icon was just hidden because this wasn't just `&.icon-event-reminder` :see_no_evil: